src/modules/visitor-space/components/DurationFilterForm/DurationFilterForm.tsx

```diff
@@ -114,7 +114,8 @@ const DurationFilterForm: FC<DurationFilterFormProps> = ({ children, className,
 
 											if (value !== form.operator) {
 												setForm({
-													duration: defaultValues.duration,
+													[SearchFilterId.Duration]:
+														defaultValues.duration,
```

Comment:
It feels a bit inconsistent that the first duration starts with a capital letter and the second duration not

Reply:
one is an enum
the second is a value

There isn't a clear enum casing convention in the code base

Which do you prefer?
I kind of like all caps to signify its a constant

SearchFilterId.DURATION